<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>449</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slider 和 QColor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSlider" name="hSliderRed">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="4" colspan="2">
      <widget class="QTextEdit" name="textColor">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>255</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="hSliderBlue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSlider" name="hSliderGreen">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSlider" name="hSliderAlpha">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>水平</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="6">
      <widget class="QSlider" name="hSliderVertical">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="6">
      <widget class="QScrollBar" name="horizontalScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="6">
      <widget class="QProgressBar" name="hProcessBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0" rowspan="2" colspan="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dial 和 LCDNumber</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" rowspan="6" colspan="2">
      <widget class="QDial" name="dial"/>
     </item>
     <item row="11" column="3" rowspan="5" colspan="2">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="11" column="5">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>LCD 进制显示</string>
       </property>
      </widget>
     </item>
     <item row="12" column="5">
      <widget class="QRadioButton" name="radioDec">
       <property name="text">
        <string>十进制</string>
       </property>
      </widget>
     </item>
     <item row="13" column="5">
      <widget class="QRadioButton" name="radioBin">
       <property name="text">
        <string>二进制</string>
       </property>
      </widget>
     </item>
     <item row="14" column="5">
      <widget class="QRadioButton" name="radioOct">
       <property name="text">
        <string>八进制</string>
       </property>
      </widget>
     </item>
     <item row="15" column="5">
      <widget class="QRadioButton" name="radioHex">
       <property name="text">
        <string>十六进制</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>161</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直</string>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>21</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>20</width>
       <height>181</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="VProcessBar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>25</width>
       <height>181</height>
      </rect>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>161</width>
      <height>222</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="bntExit">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>88</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
